microsoft-edge/toc.yml

```diff
@@ -841,6 +845,9 @@
             - name: Defining match patterns for an extension to access file URLs
               href: extensions-chromium/developer-guide/match-patterns.md  # moved from /enterprise/
 
+            - name: Create an extension that extends DevTools
```

Comment:
@Reezaali what do you think of this one? I agree that the previous version ("Create a DevTools extension") wasn't conveying the right meaning.

Reply:
Create an extension that customizes the DevTools UI

How's that?

Reply:
I personally don't like the repetition of extension and extends. What about: "Create an extension for DevTools"